app/modules/users/resources.py

```diff
@@ -369,8 +390,11 @@ def get(self, args, user):
         """
         Get Sightings for user with EDM metadata
         """
+        offset = args['offset']
+        limit = args['limit']
 
-        start, end = args['offset'], args['offset'] + args['limit']
+        start = offset
```

Comment:
having offset and limit as variables is a good idea, not sure the same can be said for start and end especially as that's not what has been done elsewhere in this PR.

Reply:
The function `user.get_sightings_json()` takes in `start` and `end`.  To prevent confusion, I made these new variable names to make things explicit that `offset/limit` does not equal `start/end`.